### ISSUE TITLE
Package request: hangover

### ISSUE THREAD

**Issue by PedroHLC** [NixOS/nixpkgs#230536]
I'm creating this issue more as research to know if someone is already working on it. Keep tabs on all partial implementations. Since adding this project at such an early stage would just increase efforts.

[**Project description**](https://github.com/AndreRH/hangover/blob/master/README.md)

Project to run (x86_64)/x86_32 Windows applications on aarch64/(ppc64le)/x86_64 Wine.

ELI5: Instead of using FEX to run Wine inside of it, you can run Wine with FEX inside of it.

**Metadata**

* homepage URL: https://github.com/AndreRH/hangover
* source URL: https://github.com/AndreRH/hangover
* license: LGPL v2.1
* platforms: Linux

**Comment by mio-19** [comment]
It would be very useful
